components/infobox/wikis/rainbowsix/infobox_team_custom.lua

```diff
@@ -0,0 +1,96 @@
+---
+-- @Liquipedia
+-- wiki=rainbowsix
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Team = require('Module:Infobox/Team')
+local Earnings = require('Module:Earnings of')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local String = require('Module:String')
+local Template = require('Module:Template')
+
+local CustomTeam = Class.new()
+
+local _CURRENT_YEAR = os.date('%Y')
+local _START_YEAR = 2015
+
+local CustomInjector = Class.new(Injector)
+
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.defineCustomPageVariables = CustomTeam.defineCustomPageVariables
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+	return team:createInfobox(frame)
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'earnings' then
+		local earnings = Earnings.team({_team.args.id or _team.pagename})
+		Variables.varDefine('earnings', earnings:gsub(',', ''))
+		if earnings == 0 then
+			earnings = nil
+		else
+			earnings = '$' .. earnings
+		end
+		return {
+			Cell{
+				name = 'Earnings',
+				content = {
+					earnings
+				}
+			}
+		}
+	elseif id == 'region' then
+		return {
+			Cell{name = 'Region', content = {_team:_createRegion(_team.args.region, _team.args.location)}}
+		}
```

Comment:
this is factually the same as the implementation in the base module since the 2nd arg (`args.location`) is not retrieved by the function

if you want that functionality i suggest to open a PR for the base module instead (unless that breaks some other wikis (sc2 does not need to be checked we kick region stuff entirely anyways))